### ISSUE TITLE
Using MPLS directly as either a source file or batch file.

### ISSUE THREAD

**Issue by skarekrow** [stax76/staxrip#39]
Hello,

Currently the tool cannot open these files. 

Since TV shows are such a PITA to rip, and then encode my current workflow has been:
```
Rip BD to BDMV structure
Mark which MPLS is which episode
Encode each episode
```
Which works great with something like Handbrake, and I can verify that these MPLS files are able to be read by both MPC-HC and VLC.

But since I prefer the workflow of Staxrip (great tool btw, thanks!) I was hoping you could fix this small hiccup, as I could manually extract each one and send the corresponding files Staxrip's way, but I'd rather not :D

Thanks again!

**Comment by stax76** [comment]
Did you try the option "Blu-ray" folder or rip with MakeMKV?

**Comment by skarekrow** [comment]
Yes thats what I've been using but since it is much slower then batch mode is for getting everything setup and then going it isn't ideal. As I already know the playlists I wish to encode. 

Bluray folder is a lot more hands on is basically my complaint for the type of mode I'm wishing, that is for tv shows. It works great when each cd movie is different, but is far too slow for X episodes. 

- Brandon

**Comment by skarekrow** [comment]
I should add that I agree MakeMKV while a valid workaround will not always remain free. I could manually do mkvtoolnix or tsmuxer and the like but that defeats the purpose a bit ;)

Since the mpls is ideal when they have many m2ts files to be concatenated since it already does that bit for us in a way. 

- Brandon

**Comment by stax76** [comment]
When I have time I'll investigate it.

**Comment by skarekrow** [comment]
Sounds great, thanks! 

- Brandon
